### PR TITLE
chore: change label

### PR DESCRIPTION
**Pull request by dodumosu**
changed export label in accordance with discussion over demo

### CODE REVIEW

File: apollo/frontend/templates/frontend/submission_list.html [hunk 1]
```diff
@@ -434,7 +434,7 @@ <h6 class="modal-title">{{ _('Nothing To Download') }}</h6>
   </button>
   <div class="dropdown-menu" aria-labelledby="exportMenuReference">
     <a class="dropdown-item" href="{{ url_for('submissions.submission_list', form_id=form.id, export='observer', **request.args) }}">{{ _('Observer') }}</a>
-    <a class="dropdown-item" href="{{ url_for('submissions.submission_list', form_id=form.id, export='observer-ts', **request.args) }}">{{ _('Observer, with section updates') }}</a>
+    <a class="dropdown-item" href="{{ url_for('submissions.submission_list', form_id=form.id, export='observer-ts', **request.args) }}">{{ _('Observer, with timestamps') }}</a>
```
Comment (takinbo) [inline apollo/frontend/templates/frontend/submission_list.html]: Can we change this to "Observer (Timestamps)"

**Review by takinbo — CHANGES_REQUESTED** [review]

**Review by takinbo — APPROVED** [review]
LGTM

**Comment by takinbo** [comment]
@dodumosu this PR needs a rebase

**Comment by dodumosu** [comment]
rebased

**Pull request merged**